packages/loader/container-loader/src/connectionStateHandler.ts

```diff
@@ -333,9 +355,11 @@ class ConnectionStateHandler implements IConnectionStateHandler {
             this.setConnectionState(ConnectionState.Connected);
         } else {
             // Adding this event temporarily so that we can get help debugging if something goes wrong.
+            // We may not see any ops due to being disconnected all that time - that's not an error!
+            const error = source === "timeout" && this.connectionState !== ConnectionState.Disconnected;
```

Comment:
FYI - I believe it is (unrelated) bug - we should not report error on timeout if we are disconnected. Even if we are connected, it's a bit moot, as we might have just connected and have not had time to observe leave op/join (i..e it might be general networking issue). But it's hard to make this check that generic, so at least covering most likely case

Reply:
Yeah I remember seeing some predictable cases here in Kusto when I was learning this code for the first time. This makes sense.